web/src/index.js

```diff
@@ -226,20 +227,54 @@ function onMediaFileSelected(event) {
     saveImageMessage(file);
   }
 }
+function onPrivateMediaFileSelected(event) {
```

Comment:
Again, I'm not really convinced we need to duplicate the logic here.

Reply:
Yaaa, you are right, since I have already populated selectedRoom each time click buttons. Have removed all the duplicate code.